.github/workflows/code-analysis.yml

```diff
@@ -28,3 +25,28 @@ jobs:
          composer install
```

Comment:
For php 5.6 you must run `composer install --no-dev`, then `composer require overtrue/phplint`

Reply:
Alternativly you can simply not run the composer install step at all and do the more simple serial run using:
` find . -type f -name '*.php' -exec php -l {} ; |grep -v "No syntax errors detected"`
or similar